pkgs/tools/filesystems/fwanalyzer/default.nix

```
@@ -0,0 +1,39 @@
+{ buildGoModule
+, fetchFromGitHub
+, lib
+, e2tools
+, mtools
+}:
+
+buildGoModule rec {
+  pname = "fwanalyzer";
+  version = "1.4.3";
+
+  src = fetchFromGitHub {
+    owner = "cruise-automation";
+    repo = pname;
+    rev = version;
+    sha256 = "1pj6s7lzw7490488a30pzvqy2riprfnhb4nzxm6sh2nsp51xalzv";
+  };
+
+  vendorSha256 = "1cjbqx75cspnkx7fgc665q920dsxnsdhqgyiawkvx0i8akczbflw";
+
+  propagatedBuildInputs = [
+    e2tools
+    mtools
+  ];
+
+  postInstall = ''
+    rm -rf $out/bin/test
```

```suggestion
    rm $out/bin/test
```

We want to notice when the file is no longer being build.

---

Thanks, removed.

---

If I don't build the not needed child package in the first place then I have not to remove it later.